### ISSUE TITLE
[Question] First and Last Date?

### ISSUE THREAD

**Issue by ethel39** [molteo-engineering-team/awesome_calendar#2]
Hello, is there a way to put first and last date range where we can pick with multiple date picker?

Thank you

**Comment by SebastienBtr** [comment]
Hello @ethel39,

I'm not sure to understand your question and what you mean by picking with multiple date pickers. There is a startDate and endDate on the calendar:

```  
/// First date of the calendar
  final DateTime? startDate;

  /// Last date of the calendar
  final DateTime? endDate;
```

But I guess this is not what you’re looking for, maybe you can try to explain a bit more what you're trying to achieve :)

**Comment by ethel39** [comment]
ah sorry, i mean for AwesomeCalendarDialog
there's no startdate and enddate parameter there

**Comment by SebastienBtr** [comment]
ah yes, you’re right, it's just hardcoded values, I'll add the fields!

**Comment by SebastienBtr** [comment]
Fixed on version `1.1.3` by https://github.com/molteo-engineering-team/awesome_calendar/commit/067b44e0c5161d14e8f61308e68ed898fedbfa1a
